### PR TITLE
Update RxAlamofire to support Swift 5 release

### PR DESCRIPTION
**Pull request by rynecheow**
- Updated podsepc version to 5.0.0
- Updated Carthage dependencies to match latest stable minor release of RxSwift and RxCocoa
- Update podspec to specify swift-version
- Update podspec to match minimum watchOS dependency of RxSwift

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,11 +25,8 @@ jobs:
           paths:
             - Carthage
       - run:
-          name: Run Tests (Swift 4)
```
Comment (freak4pc) [inline .circleci/config.yml]: The tests should still run on the latest Swift 4.2

File: .swift-version [hunk 2]
```diff
@@ -1 +1 @@
-4.0
+5.0
```
Comment (freak4pc) [inline .swift-version]: It should be the latest 4.2 we support. 
Otherwise RxAlamofire won't support Swift 4.x at all

Comment (rynecheow) [reply .swift-version]: Agreed.

File: .circleci/config.yml (hunk 1 above)
Comment (rynecheow) [reply .circleci/config.yml]: Agreed.

**Comment by freak4pc** [comment]
There's one failing test, please ping me when CI fails and I'll do a quick review. Thanks again :)

**Comment by rynecheow** [comment]
All green now :)

**Comment by freak4pc** [comment]
Just one thing @rynecheow - I'm not sure this should be RxAlamofire 5.0. What do you think ? 

There were no breaking changes here - just additive, as far as I can see.

**Comment by rynecheow** [comment]
Certainly @freak4pc! I think 4.4 would do. The only reason I bumped it to 5.0.0 is to mark Swift 5.0's release.. Haha

**Comment by freak4pc** [comment]
Can you make a PR to fix the versions perhaps? @rynecheow

**Pull request merged**